spec/deaf_grandma_spec.rb

```diff
@@ -9,10 +9,18 @@
   end
   
   it "says 'NOT SINCE 1964!' when we yell" do
-    #implement your test here
+    expect(script.speak("HELLO")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    def bye
+        3.times { script.speak("BYE") }
+    end
+
+    begin
+        bye
+    rescue SystemExit => e
```

Comment:
If there's a way to not need to `rescue SystemExit` somehow, that would be even better.